packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksEditor.tsx

```diff
@@ -38,8 +38,10 @@ interface BaseBlock {
   handleConvert?: (editor: Editor) => void | (() => React.JSX.Element);
   handleEnterKey?: (editor: Editor) => void;
   handleBackspaceKey?: (editor: Editor, event: React.KeyboardEvent<HTMLElement>) => void;
+  handleTab?: (editor: Editor) => void;
```

Comment:
maybe you could pass the event too, like handleBackspaceKey. This way you could check if shift is pressed and reverse the behavior

Reply:
I will ask @yanniskadiri to confirm and add improvement ticket to reverse the behaviour.